src/symbols.c

```diff
@@ -2289,6 +2290,32 @@ static gint get_fold_header_after(ScintillaObject *sci, gint line)
 }
 
 
+/* returns the line after following all brace match for @brace on @line */
+static gint resolve_matching_braces(ScintillaObject *sci, gint line, gint brace)
+{
+	gint pos = sci_get_position_from_line(sci, line);
+	gint line_end = sci_get_line_end_position(sci, line);
+	gint lexer = sci_get_lexer(sci);
```

Comment:
Do we really want something like that?
```C

	for (gint pos = sci_get_position_from_line(sci, line),
	          line_end = sci_get_line_end_position(sci, line),
	          lexer = sci_get_lexer(sci); pos < line_end; pos++)
```

I mean, sure in theory scoping is best, but here IMO it makes this harder to read, and the parent scope doesn't contain anything anyway, so it's not mixed with other stuff.  If it was a `size_t i`, sure, but here I'm not convinced.